Pinta.Effects/Effects/TwistEffect.cs

```diff
@@ -39,87 +40,96 @@ public override void LaunchConfiguration ()
 	#region Algorithm Code Ported From PDN
 	public override void Render (ImageSurface src, ImageSurface dst, ReadOnlySpan<RectangleI> rois)
 	{
-		float twist = Data.Amount;
+		MaterializedSettings settings = CreateSettings (dst);
```

Comment:
I'm not sure the word `Materialized` really makes sense here? I'd just call it Settings or RenderSettings

Reply:
No problem, I changed it to `RenderSettings`